src/etc/inc/plugins.inc.d/openvpn.inc

```diff
@@ -1004,6 +1004,18 @@ function openvpn_restart($mode, $settings)
 
     @unlink("/var/etc/openvpn/{$mode_id}.sock");
 
+     // has it shut down? Check the validity of the pid
```

Comment:
this is weird, like 983 already does:

killbypid("/var/run/openvpn_{$mode_id}.pid", 'TERM', true);

Reply:
I'm not taking prisoners here, hence 'KILL'. :)